src/main/java/com/ays/admin_user/controller/AdminUserRegisterApplicationController.java

```diff
@@ -131,4 +131,20 @@ public AysResponse<AdminUserRegisterApplicationCreateResponse> createRegistratio
         );
     }
 
+    /**
+     * Approves an admin user register application.
+     * Requires SUPER_ADMIN authority.
+     *
+     * @param id The id of the register application.
+     * @return A success response.
+     */
+    @PostMapping("/registration-application/{id}/approve")
+    @PreAuthorize("hasAnyAuthority('SUPER_ADMIN')")
+    public AysResponse<Void> approveRegistrationApplication(
+            @PathVariable @UUID String id) {
```

Comment:
Burası çok uzun olmadığı için alt satıra indirgemeyebiliriz

Reply:
Düzelttim.